.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -446,7 +446,7 @@ jobs:
               --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>" \
               --junit-file "cilium-junits/${{ env.job_name }} (${{ join(matrix.*, ', ') }}).xml" \
               --junit-property github_job_step="Run tests upgrade 2 (${{ join(matrix.*, ', ') }})" \
-              \$EXTRA
+              $EXTRA
```

Comment:
So this gets rid of unnecessary escaping?

Reply:
Oh, yeah. I was thinking it is just a cosmetic thing, but actually it was mistakenly escaped.